com.unity.sg2/Editor/GraphDeltaRegistry/FunctionDefinitions/StandardDefinitions/Math/Range/ClampNode.cs

```diff
@@ -9,13 +8,15 @@ internal class ClampNode : IStandardNode
         public static int Version = 1;
 
         public static FunctionDescriptor FunctionDescriptor => new(
-            Version,
             Name,
             "Out = clamp(In, Min, Max);",
-            new ParameterDescriptor("In", TYPE.Vector, Usage.In),
-            new ParameterDescriptor("Min", TYPE.Vector, Usage.In),
-            new ParameterDescriptor("Max", TYPE.Vector, Usage.In, new float[] { 1f, 1f, 1f, 1f}),
-            new ParameterDescriptor("Out", TYPE.Vector, Usage.Out)
+            new ParameterDescriptor[]
+            {
+                new ParameterDescriptor("In", TYPE.Vector, Usage.In),
+                new ParameterDescriptor("Min", TYPE.Vector, Usage.In),
+                new ParameterDescriptor("Max", TYPE.Vector, Usage.In, new float[] { 1f, 1f, 1f, 1f}),
+                new ParameterDescriptor("Out", TYPE.Vector, Usage.Out)
```

Comment:
Style question/preference-- because the parameterDescriptors are now in a typed array, would we want to encourage tech art to just use `new(...)` instead of `new ParameterDescriptor(...)` for style reasons?

Reply:
If we do, let's **not** make that change in this PR!

Reply:
Yes. We should do this. I will take a note of it and do it in a later PR.